napari_animation/interpolation.py

```diff
@@ -0,0 +1,91 @@
+import numbers
+from enum import Enum
+from functools import partial
+
+import numpy as np
+from napari._vispy.quaternion import quaternion2euler
+from scipy.spatial.transform import Rotation as R
+from scipy.spatial.transform import Slerp
+from vispy.util.quaternion import Quaternion
+
+
+def default(a, b, fraction):
```

Comment:
Can you add `Parameters` and `Returns` sections to these doc strings

Reply:
I added some to all these functions, it might be silly but just in case. ^^

Reply:
No it's really great! They should all have doc strings. We'll start building docs and sharing them online soon and then it will be really nice to have complete coverage in the standard form. Thanks!!